subsys/bluetooth/adv_prov/providers/Kconfig.tx_power

```diff
@@ -4,9 +4,22 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-config BT_ADV_PROV_TX_POWER
+menuconfig BT_ADV_PROV_TX_POWER
 	bool "TX power"
 	select BT_CTLR_TX_PWR_DYNAMIC_CONTROL
 	help
 	  Adds TX power to advertising data. Provider reads the advertising TX
-	  power from Bluetooth controller.
+	  power from Bluetooth controller and adds predefined correction value
+	  to it.
+
+if BT_ADV_PROV_TX_POWER
+
+config BT_ADV_PROV_TX_POWER_CORRECTION_VAL
```

Comment:
It's not about FEM. The observed RSSI may depend on used antenna, casing etc. (some of these parameters are not known to the SoftDevice nor MPSL).  The option is introduced to allow user to apply correction value tailored to his device to increase quality of distance measurement. This is needed e.g. in case of Fast Pair (https://developers.google.com/nearby/fast-pair/specifications/configuration#transmit_power).